lib/action_controller/serialization/context.rb

```diff
@@ -0,0 +1,13 @@
+module ActionController
+  module Serialization
+    class Context
+      attr_reader :request_url, :query_parameters, :url_helpers
```

Comment:
I think I'd rather have `url_helpers` a class method on the Context.  It feels more natural here. Also, no need to include them. you could just

```ruby
class Context
  class << self
    attr_accessor :url_helpers, :default_url_options
  end
end
```

and in the Railtie

```ruby
Context.url_helpers = Rails.application.routes.url_helpers # for now, later would like to encapsulate it to control the boundary e.g. UrlHelpers.new(Rails.application.routes.url_helpers)
Context.default_url_options =  ActionController::Base.default_url_options
```

I'm trying to keep in mind how someone using grape would configure this.  If we keep a good interface, the details of the implementation don't matter as much.  That means more composition and less just mixing things in. Alternatively, there could be a UrlHelpers module that you mix in to the Context when the railtie is called, which allows others to mix in a different helper object